src/main/groovy/ai/digital/integration/server/tasks/worker/WorkerOverlaysTask.groovy

```diff
@@ -0,0 +1,55 @@
+package ai.digital.integration.server.tasks.worker
+
+import ai.digital.integration.server.domain.Worker
+import ai.digital.integration.server.util.WorkerUtil
+import org.gradle.api.Action
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Copy
+
+import static ai.digital.integration.server.constant.PluginConstant.PLUGIN_GROUP
+
+class WorkerOverlaysTask extends DefaultTask {
+    static NAME = "workerOverlays"
+
+    static PREFIX = "worker"
+
+    static boolean shouldUnzip(File file) {
```

Comment:
Let's create a util for that, it is copy pasted many times already

Reply:
Moved everything that is repeating to the OverlaysUtil.